src/queueUI/Line.java

```diff
@@ -0,0 +1,75 @@
+package queueUI;
+
+import javax.swing.*;
+
+
+public class Line {
+
+    private Node startQueue, endQueue;
+    String queue = "";
+
+    // create constructor
+    public Line() {
+        startQueue = null;
+        endQueue = null;
+    }
+
+    // method to know if the queue is empty.
+    public boolean EmptyQueue() {
+        if (startQueue == null) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    // method to insert information in the queue
+    public void Insert(int inputData) {
+        Node recentNode = new Node();
+        recentNode.inputData = inputData;
+        recentNode.after = null;
```

Comment:
It is better to initialize the attributes of an object in the constructor method. For example:
```java
public class Node {
    int inputData;
    Node after;

    public Node(int inputData, Node after) { # take both inputData and after as parameters
        this.inputData = inputData;
        this.after = after;
    }
}
```

Then, when you create the object you can do:
```java
Node recentNode = new Node(inputData, null);
```

---

If the after attribute will always be null when the object is created, you could do it this way:

```java
public class Node {
    int inputData;
    Node after;

    public Node(int inputData) { # we no longer take after as parameter
        this.inputData = inputData;
        this.after = null;
    }
}
```

Then, when you create the object you can do:
```java
Node recentNode = new Node(inputData);
```